doc/Type/IO/Socket/Async.pod6

```diff
@@ -12,13 +12,26 @@ server and the client side.
 Here is the equivalent example to that in L<IO::Socket::INET> of a simple
 echo server that listens on port 3333:
 
-=begin code
-use v6.c;
-
-react {
-    whenever IO::Socket::Async.listen('localhost', 3333) -> $conn {
-        whenever $conn.Supply(:bin) -> $buf {
-            await $conn.write: $buf
+=begin code :skip-test<IO>
+loop {
+    react {
+        whenever IO::Socket::Async.listen('0.0.0.0', 8080) -> $conn {
+            react { 
+                whenever $conn.Supply.lines {
+                    say .Str;
+
+                    { await $conn.print(.Str ~ "\n") } for 
```

Comment:
Hm, I wonder if this is actually needed, or do we only have to await for the last print?